### PR TITLE
Fix for #37 Improve code readability by removing Yoda style conditions

### LINKED ISSUE #37: Improve code readability by removing Yoda style conditions
## Description
Yoda conditions are prevalent in the code base [Yoda](https://knowthecode.io/yoda-conditions-yoda-not-yoda). While these are not technically wrong, it's been brought up that we might want to change this throughout the code base.

## Additional info
Refer to discussion here: https://github.com/a-rmz/JEasyCrypto/pull/33#pullrequestreview-69778606

### PR DESCRIPTION
**Pull request by Lrantala**
#### What does this PR do?
Fixes the Yoda-like conditions that were mentioned in #37.

#### Where should the reviewer start?
CryptoClient.java and CryptoServer.java were the only affected files, so there.

#### What are the relevant tickets?
Closes #37

### CODE REVIEW

File: JEasyCryptoClient/src/CryptoClient.java [hunk 1]
```diff
@@ -68,7 +68,7 @@ public void run() {
 					break;
 				}
 				}
-			} while (choice != QUIT_MENU);
+			} while (QUIT_MENU != choice);
```
Comment (risju62) [inline JEasyCryptoClient/src/CryptoClient.java]: Seems Yoda to me, should be like variable != constant.

Comment (Lrantala) [reply JEasyCryptoClient/src/CryptoClient.java]: This fixed is now. :D

File: JEasyCryptoClient/src/CryptoClient.java [hunk 2]
```diff
@@ -68,7 +68,7 @@ public void run() {
 					break;
 				}
 				}
-			} while (choice != QUIT_MENU);
+			} while (choice != QUIT_MENU );
```
Comment (a-rmz) [inline JEasyCryptoClient/src/CryptoClient.java]: Just this extra space.

**Comment by a-rmz** [comment]
I'm also curious, this PR includes two commits made some time ago by me. From which branch did you fork and when?

**Comment by Lrantala** [comment]
I did the sync after you had posted we should do that. And then resumed normally with the work.

File: JEasyCryptoClient/src/CryptoClient.java (hunk 2 above)
Comment (Lrantala) [reply JEasyCryptoClient/src/CryptoClient.java]: Done. With a bit of messing up with Git. :D But now it should be okay.

Comment (a-rmz) [reply JEasyCryptoClient/src/CryptoClient.java]: Great, well done! 🎉 🌮

**Comment by a-rmz** [comment]
:shipit:

**Pull request merged**